utils.py

```diff
@@ -246,3 +249,16 @@ def check_software_installed(list_software=['sct']):
             logging.error("'{}' is not installed. Please install it before using this program.".format(software))
             install_ok = False
     return install_ok
+
+
+def create_empty_mask(fname, fname_label):
+    """
+    Create empty mask from reference image
+    :param fname: path to reference image
+    :param fname_label: path to output mask under derivatives
+    """
+    img = nib.load(fname)
+    data = np.zeros(img.shape)
+    img_mask = nib.Nifti1Image(data, img.affine, img.header)
+    nib.save(img_mask, fname_label)
+    print("Empty mask created: {}".format(fname_label))
```

Comment:
Maybe edit this to say `Empty mask created at: {}".format(fname_label)`

Reply:
Okay! Done in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/b1f341c420f073350ad2bd45184af8a6f6e03785